autotst/calculator/gaussian.py

```diff
@@ -57,6 +57,8 @@ def __init__(self,
                      "mem": "5GB",
                      "nprocshared": 20,
                  },
+                 scratch=".",
```

Comment:
I don't believe `self.scratch` is used anywhere so why is it an attribute?

Reply:
I removed self.scratch in commit 3e8d36a